sdk/state/close.go

```diff
@@ -63,21 +57,15 @@ func amountToResponder(balance int64) int64 {
 	return 0
 }
 
-func (c *Channel) CoordinatedCloseTx() (*txnbuild.Transaction, error) {
-	txClose, err := c.makeCloseTx(c.latestAuthorizedCloseAgreement.Details, 0, 0)
-	if err != nil {
-		return nil, err
-	}
-	return txClose, nil
-}
-
 // ProposeCoordinatedClose proposes a close transaction to be submitted immediately.
 // This should be used when participants are in agreement on the final txClose parameters, but would
 // like to submit earlier than the original observation time.
 func (c *Channel) ProposeCoordinatedClose() (CloseAgreement, error) {
 	d := c.latestAuthorizedCloseAgreement.Details
+	d.ObservationPeriodTime = 0
+	d.ObservationPeriodLedgerGap = 0
 
-	txCoordinatedClose, err := c.makeCloseTx(d, 0, 0)
+	_, txCoordinatedClose, err := c.CloseTxs(d)
```

Comment:
It probably seems a bit non-performant to build all transactions here instead of only the close transaction, but that won't matter soon when we implement #80, and even if we weren't doing that, a coordinated close only happens once in a channels lifetime and so the extra work here isn't really that important.